tensorflow_addons/layers/python/normalizations_test.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+from tensorflow_addons.layers.python.normalizations import GroupNormalization,LayerNormalization,InstanceNormalization
+import numpy as np
+import scipy as scipy
+import tensorflow as tf
+from tensorflow import keras as keras
+from tensorflow.python.training.rmsprop import RMSPropOptimizer
+from tensorflow.python.platform import test
+from tensorflow.python.framework import test_util as tf_test_util
```

Comment:
Check [PEP 0008 import](https://www.python.org/dev/peps/pep-0008/#imports) section and fix those imports here.